src/main/resources/com/playground/playground/graph-system-view.fxml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.chart.NumberAxis?>
+<?import javafx.scene.chart.ScatterChart?>
+<?import javafx.scene.control.CheckBox?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.VBox?>
+
+<VBox prefHeight="457.0" prefWidth="485.0" xmlns="http://javafx.com/javafx/20.0.1" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.playground.playground.GraphSystemController">
+   <VBox prefHeight="462.0" prefWidth="485.0">
+      <children>
+          <ScatterChart fx:id="neuralNetwork" maxHeight="-Infinity" maxWidth="-Infinity" prefHeight="400.0" prefWidth="400.0" styleClass="neural-network-text">
+              <xAxis>
+                  <NumberAxis fx:id="yAxis" lowerBound="-5.5" side="BOTTOM" upperBound="5.5" />
+              </xAxis>
+              <yAxis>
+                  <NumberAxis lowerBound="-5.5" side="RIGHT" upperBound="5.5" fx:id="xAxis" />
```

Comment:
The bounds for the axes should be from -10 to 10 to be consistent with our dataset here too.
```suggestion
                  <NumberAxis lowerBound="-10" side="RIGHT" upperBound="10" fx:id="xAxis" />
```

Reply:
I'll change that.